cypress/e2e/pages/articles/index.cy.js

```diff
@@ -1,16 +1,240 @@
+  },
+  {
+    path: '/serbian/articles/c805k05kr73o/cyr',
+    runforEnv: ['local'],
+    service: 'serbian',
+    variant: 'cyr',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/serbian/articles/c805k05kr73o/lat',
+    runforEnv: ['local'],
+    service: 'serbian',
+    variant: 'lat',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/zhongwen/articles/c3xd4x9prgyo/simp',
+    runforEnv: ['local'],
+    service: 'zhongwen',
+    variant: 'simp',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/zhongwen/articles/c3xd4x9prgyo/trad',
+    runforEnv: ['local'],
+    service: 'zhongwen',
+    variant: 'trad',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+];
+
+const nonSmokeCanonicalTestSuites = [
+  {
+    path: '/afaanoromoo/articles/c4g19kgl85ko',
+    runforEnv: ['test'],
+    service: 'afaanoromoo',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/afrique/articles/cz216x22106o',
+    runforEnv: ['test'],
+    service: 'afrique',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/azeri/articles/cv0lm08kngmo',
+    runforEnv: ['live'],
+    service: 'azeri',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/japanese/articles/cdd6p3r9g7jo',
+    runforEnv: ['test'],
+    service: 'japanese',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/japanese/articles/cj4m7n5nrd8o',
+    runforEnv: ['live'],
+    service: 'japanese',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/kyrgyz/articles/c414v42gy75o',
+    runforEnv: ['live'],
+    service: 'kyrgyz',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/kyrgyz/articles/c41knv20gk7o',
+    runforEnv: ['live'],
+    service: 'kyrgyz',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/kyrgyz/articles/cpgx4k72wv4o',
+    runforEnv: ['live'],
+    service: 'kyrgyz',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/mundo/articles/ce7p1pw7165o',
+    runforEnv: ['live'],
+    service: 'mundo',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+
+  {
+    path: '/nepali/articles/c16ljg1v008o',
+    runforEnv: ['live'],
+    service: 'nepali',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+  {
+    path: '/ukrainian/articles/c8zv0eed9gko',
+    runforEnv: ['live'],
+    service: 'ukrainian',
+    tests: [...testsForAllPagesCrossPlatform, canonicalTests],
+  },
+];
+
+/**
+ * TODO: Determine whether when running scheduled E2Es, should we run the smoke URLs too?
+ * Or should we only run the non-smoke tests - and leave the smoke tests for the Simorgh CD pipeline
+ */
```

Comment:
Need a second opinion on this

Reply:
I think if we're thinking of smoke tests as things to catch big fails early on as they just test critical functionality, i'd be tempted to just run them as part of the simorgh CD pipeline i.e. only when changes are made? As then they'd be a thing that determines if we're happy to go ahead with further testing as part of the scheduled E2E's. Fair few definitions of smoke testing online seem to define it as just preliminary testing , which i think fits more in the CD pipeline.

Reply:
Yes, that's a good shout. The smoke tests will have already run on the pipeline (and would have given us confidence that things are OK), so we don't need to re-run them again on the scheduled E2Es. I am happy with this rubber-🦆-ing so I can go and make the changes now 😄